cli/build.rs

```diff
@@ -0,0 +1,36 @@
+use std::process::{Command, ExitStatus};
+
+fn main() {
+    set_env("GIT_HASH", "git", &["rev-parse", "HEAD"]);
```

Comment:
As I wrote before: add a prefix with short application name to the name of the environment variable. `GIT_HASH` is too common, so there is a possibility of name collision